app/views/welcome/index.html.erb

```diff
@@ -1,40 +1,106 @@
-<h1>RideBoard.app</h1>
+<%# Navbar%>
+<nav class="welcome">
+  <div class="nav-wrapper">
+    <a href="/" class="logo">RideBoard.app</a>
+    <a href="#" data-target="mobile-sidenav" class="sidenav-trigger"><i class="material-icons">menu</i></a>
+    <ul class="right hide-on-med-and-down">
+      <% if content_for? :rides_link %>
+        <li><%= yield :rides_link %></li>
+      <% end %>
 
-<div>
-<%= link_to "I am a passenger", passenger_root_path %>
-</div>
-<div>
-<%= link_to "I am a driver", driver_root_path %>
-</div>
+      <% if user_signed_in? && current_user.admin? %>
+        <li><%= link_to 'Locations', locations_path %></li>
+      <% end %>
 
-<section>
-  <h2>What is this app?</h2>
+      <% if user_signed_in? %>
+        <li><%= link_to "Sign Out: #{user_display_name(current_user)}",
+                destroy_user_session_path, method: :delete %></li>
+      <% else %>
+        <li><%= link_to 'Sign In', new_user_session_path %></li>
+        <li><%= link_to 'Sign Up', new_user_registration_path %></li>
+      <% end %>
+
+      <% if content_for? :other_app_link %>
+        <li><%= yield :other_app_link %></li>
+      <% end %>
+    </ul>
+  </div>
+</nav>
 
+<ul class="sidenav" id="mobile-sidenav">
+  <% if content_for? :rides_link %>
+    <li><%= yield :rides_link %></li>
+  <% end %>
+
+  <% if user_signed_in? && current_user.admin? %>
+    <li><%= link_to 'Locations', locations_path %></li>
+  <% end %>
+
+  <% if user_signed_in? %>
+    <li><%= link_to "Sign Out: #{user_display_name(current_user)}",
+            destroy_user_session_path, method: :delete %></li>
+  <% else %>
+    <li><%= link_to 'Sign In', new_user_session_path %></li>
+    <li><%= link_to 'Sign Up', new_user_registration_path %></li>
+  <% end %>
+</ul>
+<%# Call to Action%>
+<div class="row container call_to_action">
+  <h1 class="white-text">Are you a driver or passenger?</h1>
+  <h2 class="white-text">RideBoard lets you find drivers or passengers to share a ride with, wherever.</h2>
+  <div class="col s12 m6 l4">
+    <div class="card z-depth-3">
+      <div class="card-image">
+        <img src="/assets/images/driver.jpg">
```

Comment:
This also isn't working in production.  I think the prod build attaches a hash to the file name.  Best to use an [asset tag](https://api.rubyonrails.org/classes/ActionView/Helpers/AssetTagHelper.html#method-i-image_tag).